Runtime/Scripts/GltfAssetBase.cs

```diff
@@ -122,6 +127,16 @@ public class GltfAssetBase : MonoBehaviour
             return importer?.GetMaterial(index);
         }
 
+        /// <summary>
+        /// Returns an imported glTF's animation clips.
+        /// Note: Asset has to have finished loading before!
+        /// </summary>
+        /// <returns>glTF animation array if it was loaded successfully, null otherwise.</returns>
+        public UnityEngine.AnimationClip[] GetAnimationClips() {
+        	Debug.Log("Importer "+importer);
```

Comment:
Please remove the `Log` and make sure indentation is consistent to the rest of the file.